doc/source/cluster/kubernetes/user-guides/gcp-gke-tpu-cluster.md

```diff
@@ -0,0 +1,98 @@
+(kuberay-gke-tpu-cluster-setup)=
+
+# Start Google Cloud GKE Cluster with TPUs for KubeRay
+
+See the [GKE documentation](<https://cloud.google.com/kubernetes-engine/docs/how-to/tpus>) for full details, or continue reading for a quick start.
+
+## Step 1: Create a Kubernetes cluster on GKE
+
+Run all following commands on your local machine or on the [Google Cloud Shell](https://cloud.google.com/shell). If running from your local machine, you need to install the [Google Cloud SDK](https://cloud.google.com/sdk/docs/install).
+
+### GKE Autopilot
+
+Create an Autopilot GKE cluster with the [Ray Operator](https://cloud.google.com/kubernetes-engine/docs/add-on/ray-on-gke/concepts/overview) enabled in the us-central2-b compute region:
+
+```sh
+gcloud container clusters create-auto kuberay-tpu-cluster \
+    --enable-ray-operator \
+    --cluster-version=CLUSTER_VERSION \
```

Comment:
since cluster version is 1.30, for now we need to specify ` --release-channel rapid`

Reply:
This flag does not appear to be needed for the Standard cluster CLI command (ie. `gcloud container clusters create ...`), I removed the Autopilot commands in [d1c3338](https://github.com/ray-project/ray/pull/45924/commits/d1c33386da6d1bfa18dfa76ec98fa157af93d05e) based on the offline discussion about ephemeral storage limits, but will add these instructions back in with the `--release-channel rapid` flag once that's resolved.